### PR TITLE
【#issu-49】Update Dockerfile.in

### PR DESCRIPTION
**Pull request by ritel126**
It shuld be set. Or the agent can't find corev1api of kubernetes!

### CODE REVIEW

**Comment by hightall** [comment]
Please create an issue related, and set title to [#issue-no] xxx

File: build_image/docker/agent/kubernetes/Dockerfile.in [hunk 1]
```diff
@@ -10,6 +10,6 @@ COPY src/agent/kubernetes-agent/src /app
 WORKDIR /app
 
 ENV KUBECONFIG /app/.kube/config
-ENV PYTHONPATH /app:$PATHONPATH
+ENV PYTHONPATH /app:/usr/local/lib/python3.6:/usr/local/lib/python3.6/lib-dynload:/usr/local/lib/python3.6/site-packages
```
Comment (hightall) [inline build_image/docker/agent/kubernetes/Dockerfile.in]: It's better get python version through PYTHON_VERSION env, and set here ${YOUR_PYTHON_VERSION}

**Comment by yeasy** [comment]
@ritel126 Please correct the DCO checking: https://github.com/hyperledger/cello/pull/44/checks?check_run_id=274198454.

**Comment by litong01** [comment]
Need your signature. Please amend it.

**Comment by litong01** [comment]
Closed this and create a new PR after discussions in the meeting

**Pull request closed without merging**